addons/mrp/models/mrp_workorder.py

```diff
@@ -237,7 +237,8 @@ def unlink(self):
 
     @api.depends('production_id.product_qty', 'qty_produced')
     def _compute_is_produced(self):
-        for order in self:
+        self.is_produced = False
```

Comment:
in master it is now possible to set a value on a recordset ?

Reply:
Yes, this is one of the benefit of the 13.0 changes

https://github.com/odoo/odoo/blob/db2e6bc600364c00e8a1d528d4b17ae052ecd41e/odoo/fields.py#L1098-L1103